pyproject.toml

```diff
@@ -24,7 +24,7 @@ dependencies = [
     # web server
     "tornado>=6.1,<7",
     # python 3.8 compatibility
-    "typing_extensions>=4.4.0",
+    "typing_extensions>=4.4.0; python_version < \"3.9\"",
```

Comment:
```suggestion
    "typing_extensions>=4.4.0; python_version < \"3.10\"",
```

Reply:
Oh great I will investigate later today :) Thanks for reviewing!

Reply:
That's true. The only place where we need `typing_extensions` is in `core/web_component.py` to import `TypeAlias`.